yarn.lock

```diff
@@ -3503,6 +3503,23 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@eslint/eslintrc@npm:^1.3.0":
```

Comment:
What's this do for us that we couldn't do before? 👀

Reply:
I have no idea what https://github.com/eslint/eslintrc is, but definitely not something I would have installed; unless it came in as some sub-dependency of something else.